pySDC/implementations/problem_classes/HeatEquation_Chebychev.py

```diff
@@ -0,0 +1,350 @@
+import numpy as np
+from scipy import sparse as sp
+
+from pySDC.core.problem import Problem
+from pySDC.implementations.datatype_classes.mesh import mesh
+from pySDC.implementations.problem_classes.generic_spectral import GenericSpectralLinear
+
+
+class Heat1DChebychev(GenericSpectralLinear):
```

Comment:
Please put some class documentation (mentioning in particular that problem is solved on [0,1] since it's hard-coded ...